backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5735100_sys_C_BPartner_Adv_Search_ExternalId.sql

```diff
@@ -0,0 +1,42 @@
+-- Run mode: SWING_CLIENT
+
+
+
+
+-- Extra precaution
+
+
+create table backup.C_Bpartner_Adv_Search_Columns_27092024 AS SELECT * FROM ad_table WHERE ad_table_ID = 541761;
+
+
+-- DROP  clumn in case it exists on a private repo
+
+
+DELETE FROM AD_Column WHERE ad_table_id = 541761 AND Columnname ilike 'ExternalId';
+
+-- 2024-09-27T07:58:24.661Z
+/* DDL */ SELECT public.db_alter_table('C_BPartner_Adv_Search','ALTER TABLE public.C_BPartner_Adv_Search DROP COLUMN IF EXISTS ExternalId')
```

Comment:
hmmm 
* when externalId column exist?
* what will happen with the value of existing externalId?